inc/composer/class-plugin.php

```diff
@@ -54,6 +54,11 @@ public function install_files() {
 		$source = $this->composer->getConfig()->get( 'vendor-dir' ) . '/altis/cms';
 		$dest   = dirname( $this->composer->getConfig()->get( 'vendor-dir' ) );
 
+		// Bail if altis/cms isn't installed, unpredictable environment.
+		if ( file_exists( $source . '/index.php' ) ) {
```

Comment:
Presumably, this should be
```suggestion
		if ( ! file_exists( $source . '/index.php' ) ) {
```